libsolidity/ast/Types.cpp

```diff
@@ -1378,12 +1378,25 @@ bool StringLiteralType::operator==(Type const& _other) const
 
 std::string StringLiteralType::toString(bool) const
 {
-	size_t invalidSequence;
+	auto isPrintableASCII = [](string const& s)
+	{
+		for (auto c: s)
+		{
+			if (static_cast<unsigned>(c) <= 0x1f || static_cast<unsigned>(c) >= 0x7f)
+				return false;
+		}
```

Comment:
```suggestion
			if (static_cast<unsigned>(c) <= 0x1f || static_cast<unsigned>(c) >= 0x7f)
				return false;
```

Reply:
if you can apply this one, I think parenthesis are really not needed

Reply:
I don't care, I think a while ago we preferred to always have braces around for loops if they had branching within, but obviously this case has no such risk.

Reply:
No, we never had braces unless the compiler warned about it.